.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: bash
+
+env:
+  global:
+    - REPO="${TRAVIS_REPO_SLUG#*/}"
+    - OWNER=plus3it
+    - RELEASE_BRANCH=master
+    - RELEASE_VERSION=$(grep -E '^current_version' $TRAVIS_BUILD_DIR/.bumpversion.cfg | sed 's/^.*= //')
+
+jobs:
+  include:
+    - stage: test
+      name: "EditorConfig Lint"
+      install:
+        - npm install -g eclint
+        - eclint --version
+      language: node_js
+      node_js: 8
+      script:
+        - make eclint/lint
```

Comment:
Add an `.editorconfig` file, can use the one from the cookiecutter-tfmodule project.